integration-tests/tests/test_integration.py

```diff
@@ -430,24 +444,27 @@ def agent_hotload(self, sync_accounts, agent_keystore, agent_keystore_path, tmp_
             yield
 
     @pytest_asyncio.fixture
-    async def client(self, agent):
+    async def client(self, agent, tmp_path):
```

Comment:
why do you need to pass tmp_path through here? seems like this function doesn't use it. I think you could add tmp_path as an argument directly to the tests that need it below